src/Relation/Nullary/Decidable.agda

```diff
@@ -60,8 +59,10 @@ module _ {a₁ a₂ b₁ b₂} {A : Setoid a₁ a₂} {B : Setoid b₁ b₂}
 -- A lemma relating True and Dec
 
 True-↔ : (dec : Dec P) → Irrelevant P → True dec ↔ P
```

Comment:
This looks like a duplicate of `True↔` from src/Function/Related/TypeIsomorphisms.agda !

Reply:
Isn't the latter deprecated (or about to be)?

Reply:
Re: duplication
I had wondered about this, but there is no new content here from me. Actually, I wasn't at all sure if the arrow was the same... and in the proofs, one uses `mk<funny arrow>`, while the other uses `inverse`... IIRC

Re: deprecation
Again I stand in ignorance. I was blindly chasing dependencies, rather than looking up at the bigger picture to 'see what is going on'. 

Either way, I'd suggest merging my commits, and then figuring out the downsteam deprecation/dependencies, but if you want me to look harder, I guess I can, but not before the weekend...

Reply:
Re: possible deprecation
Can you link the/a relevant issue/PR please?

Reply:
None of the two modules involved seem to be deprecated. I agree it's mostly orthogonal to this PR,
it's just something I noticed because the PR was hitting both definitions.

Reply:
Indeed, and I think it would be better to  redefine this lemma to be its counterpart. Reason? complex `import` dependencies wrt relevant/irrelevant elimination of \bot...

... but I'm abandoning this PR, because whatever has happened recently, this is now too broken to want to fix, especially for v2.0.